src/plcc.py

```diff
@@ -1014,6 +1036,103 @@ def semFinishUp():
         print('  {}.java'.format(cls))
 
 
+
+
+def sempy(nxt):
+    global stubs, argv
+    # print('=== python semantic routines')
+    if not getFlag('python_semantics'):
+        sempyFinishUp()
+    for line in nxt:
+        line = line.strip()
+        if line[:7] == 'include':
```

Comment:
We're not going to allow the classic 'include' in the new 4th section. So let's remove this section.